cli/agent.go

```diff
@@ -480,6 +488,14 @@ func (r *RootCmd) workspaceAgent() *serpent.Command {
 			Description: "Allow the agent to automatically detect running devcontainers.",
 			Value:       serpent.BoolOf(&devcontainersEnabled),
 		},
+		{
+			Flag:        "experimental-connection-reports-enable",
+			Hidden:      true,
+			Default:     "false",
+			Env:         "CODER_AGENT_EXPERIMENTAL_CONNECTION_REPORTS_ENABLE",
+			Description: "Enable experimental connection reports.",
+			Value:       serpent.BoolOf(&experimentalConnectionReports),
+		},
```

Comment:
Should this be something that gets set as a deployment-wide experiment and is automatically set in the agent manifest?

Reply:
The reason this is an (temporary) experiment is simply to not mess up the audit log UI until it is updated. I'm not aware that this needs to be placed behind an experiment deployment wide.